logic.js

```diff
@@ -0,0 +1,108 @@
+const Froger = () => {
+    const _frogs = [];
+    let _currentLevel = '-';
+    let _timer = '-';
+    let _timerId = null;
+    let _frogsLeft = '-';
+    let _isGameOver = false;
+    let _timerColor = 'black';
+
+    const getFrogs = () => _frogs;
+    const getLevel = () => _currentLevel;
+    const getTime = () => _timer;
+    const getFrogsLeft = () => _frogsLeft;
+    const getTimerColor = () => _timerColor;
+    const getIsGameOver = () => _isGameOver
+
+    const getRandomColor = () => {
+        var letters = '0123456789ABCDEF';
+        var color = '#';
+        for (var i = 0; i < 6; i++) {
+            color += letters[Math.floor(Math.random() * 16)];
+        }
+        return color;
+    }
+
+    const toggleRed = () => {
```

Comment:
the name if this function could be - toggleTimerColor - because it's not just toggle the timer to red (it also toggle it to black).